### PR TITLE
Move all pdf generation to 'app'

### PR DESCRIPTION
**Pull request by Martouta**
Move everything from "lib/pdf" to "app/lib/pdf".
Remove the old logo. Use the new one (without repeating it).
Fix CodeClimate issues.

### CODE REVIEW

**Comment by duduribeiro** [comment]
I opened https://github.com/3scale/porta/pull/1440 with this and also to fix some CodeClimate issues for it

**Comment by hallelujah** [comment]
Closing in favor of #1440

**Comment by Martouta** [comment]
1440 was merged to this one and not to master.

File: test/unit/pdf/finance/invoice_generator_test.rb [hunk 1]
```diff
@@ -59,7 +59,7 @@ class Pdf::Finance::InvoiceGeneratorTest < ActiveSupport::TestCase
 
   # for development purposes
   def see_in_file(content)
-    f = File.new('/home/jakub/Desktop/a.pdf','w')
+    f = File.new('/home/myusername/Desktop/a.pdf','w')
```
Comment (hallelujah) [inline test/unit/pdf/finance/invoice_generator_test.rb]: 🙄 just remove this 🔪

File: app/lib/pdf/data.rb [hunk 2]
```diff
@@ -57,30 +56,26 @@ def top_users
...
+      @service.published_plans.map { |plan| Format.prep_td([plan.name, plan.cinstances.count]) }
     end
 
     def metrics
-      data_source = Stats::Service.new(@service)
-
       data = @source.usage_progress_for_all_metrics(@options)[:metrics]
       data.inject([]) do |row, (metric, stats)|
-        row << Format.prep_td_with_negation([metric[:name], metric[:data][:total], "#{"%0.2f" % metric[:data][:change]} %"])
+        row << Format.prep_td_with_negation([metric[:name], metric[:data][:total], "#{format('%0.2f', metric[:data][:change])} %"])
```
Comment (hallelujah) [inline app/lib/pdf/data.rb]:
```suggestion
        percentage = '%0.2f %' % metric[:data][:change]
        row << Format.prep_td_with_negation([metric[:name], metric[:data][:total], percentage])
```

**Pull request merged**